File: src/main/java/com/fish/KexieOpenJudgeApplication.java
package com.fish;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("com.fish.mapper")
public class KexieOpenJudgeApplication {
    public static void main(String[] args) {
        SpringApplication.run(KexieOpenJudgeApplication.class, args);
    }
}

File: src/main/kotlin/com/fish/KexieOpenJudgeApplication.kt
package com.fish

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.EnableAspectJAutoProxy

@SpringBootApplication
@MapperScan("com.fish.mapper")
class KexieOpenJudgeApplication
fun main(args : Array<String>) {
    runApplication<KexieOpenJudgeApplication>(*args)
}
